lib/repo-audit/checks/at_least_one_file_check.rb

```diff
@@ -0,0 +1,22 @@
+module RepoAudit::Checks
+  class AtLeastOneFileCheck < BaseCheck
+    include RepoAudit::RepositoryContent
+
+    register_check :at_least_one_file_check
+
+    attr_accessor :filenames
+
+    def run(repo)
+      at_least_one_file_exists?(repo) ? success : failure
+    end
+
+    private
+
+    def at_least_one_file_exists?(repo)
```

Comment:
Actually, scratch this. 

It's a private method, and in the context it's being used, the result is only used to pick between success and failure branches, so coercing to a boolean is probably overkill.